packages/server/graphql/mutations/addPokerTemplateScaleValue.ts

```diff
@@ -0,0 +1,71 @@
+import {GraphQLID, GraphQLInt, GraphQLNonNull} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import standardError from '../../utils/standardError'
+import TemplateScaleInput from '../types/TemplateScaleInput'
+import AddPokerTemplateScaleValuePayload from '../types/AddPokerTemplateScaleValuePayload'
+
+const addPokerTemplateScaleValue = {
+  description: 'Add a new scale value for a scale in a poker template',
+  type: AddPokerTemplateScaleValuePayload,
+  args: {
+    scaleId: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    scaleValue: {
+      type: new GraphQLNonNull(TemplateScaleInput)
+    },
+    index: {
+      type: GraphQLInt
+    }
+  },
+  async resolve(
+    _source,
+    {templateId, scaleId, scaleValue, index},
+    {authToken, dataLoader, socketId: mutatorId}
+  ) {
+    const r = await getRethink()
+    const operationId = dataLoader.share()
+    const subOptions = {operationId, mutatorId}
+    const template = await r
+      .table('MeetingTemplate')
+      .get(templateId)
+      .run()
+    const viewerId = getUserId(authToken)
+
+    // AUTH
+    if (!template || !isTeamMember(authToken, template.teamId) || !template.isActive) {
+      return standardError(new Error('Team not found'), {userId: viewerId})
+    }
+
+    // VALIDATION
```

Comment:
we need some validation for the scaleValue here.

- the label should be a max of 2 chars
- the value should be a number 0-99
- the color should start with a # and be followed by 6 hex numbers normalized to uppercase to keep it all clean)

Reply:
one thing to note:
there exist some special cards like ❓  and 🚫 .
❓ should have a value of 2^31 -1
🚫  should have a value of -1
we give them those values for the sort order.
we should make exceptions for these 2 values.